### PR TITLE
[plugin.video.catchuptvandmore] 0.2.11

### PR DESCRIPTION
**Pull request by SylvainCecchetto**
### Description

Addon update

[Version 0.2.11]
[Fix] Live TV - NRJ 12 (FR) - Thank you Psychose
[Fix] Live TV - Télé Quebec (CA)
[Add] Live TV - Azur TV (FR)
[Fix] Catch-up TV - Arte (WO) - some updates (description, erros, etc...)
[Add] Live TV - BIP TV (FR)
[Fix] Live TV - Tébéo (FR)
[Fix] Catch-up TV and Live TV - Blaze TV (UK)
[Add] Live TV - Grand Lille TV (FR)
[Add] Catch-up TV - J-One (by using videos from dailymotion)
[Add] Live TV - Grand Littoral TV (FR)
[Add] Live TV - La Chaine Normande (FR)
[Remove] Catch-up TV - Onzeo (FR)
[Add] Live TV - Sport En France (FR) - (La chaine du mouvement sportif)
[Fix] Catch-up TV and Live TV - La7 (IT)
[Fix] Live TV - Paramount Channel (IT)
[Add] Live TV - Provence Azur TV (FR)
[Fix] Website - Allocine (FR) - Video without title
[Add] Catch-up TV and Live TV - France 3 Région - Nouvelle-Aquitaine (FR)
[Add] Catch-up TV and Live TV - TébéSud (FR)
[Fix] Catch-up TV - NHK World (WO)
[Add] Live TV - MaTélé (FR)

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.catchuptvandmore/resources/lib/channels/fr/tebeo.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 # TODO
 
-URL_ROOT = 'http://www.tebeo.bzh'
+URL_ROOT = 'http://www.%s.bzh'
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/channels/fr/tebeo.py]: Why this change? This makes the root URL so generic it's hard to check.

**Comment by wwark** [comment]
Hi @enen92,
I work with @SylvainCecchetto for this plugin.
We made this change that because we implement http://www.tebesud.bzh/ using the same code as https://www.tebeo.bzh/. Both websites are quite similar.
In the code, we replace %s by item_id. The item_id is the name of the channel. In this case the name is same as the hostname of the website (tebesud or tebeo). 
During the HTTP GET, we used this code "resp = urlquick.get(URL_REPLAY % item_id)"
URL_REPLAY is construct like below:
URL_REPLAY = URL_ROOT + '/le-replay'
URL_ROOT = 'http://www.%s.bzh'
It is the same for the live TV.
Thanks,
Wwark.

**Comment by Rechi** [comment]
@SylvainCecchetto this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.
